ogm/documentation/getting-started.adoc

```diff
@@ -236,33 +238,18 @@ tm.commit();
 
 emf.close();
 
-private static final String JBOSS_TM_CLASS_NAME = "com.arjuna.ats.jta.TransactionManager";
-
-public static TransactionManager getTransactionManager() throws Exception {
-    Class<?> tmClass = Main.class.getClassLoader().loadClass(JBOSS_TM_CLASS_NAME);
-    return (TransactionManager) tmClass.getMethod("transactionManager").invoke(null);
-}
 ----
 
-[NOTE]
-====
-Some JVM do not handle mixed IPv4/IPv6 stacks properly (older
-http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=7144274[Mac OS X JDK in particular]),
-if you experience trouble starting the Infinispan cluster,
-pass the following property: +-Djava.net.preferIPv4Stack=true+
-to your JVM or upgrade to a recent JDK version.
-jdk7u6 (b22) is known to work on Max OS X.
-====
-
-A working example can be found in Hibernate OGM's distribution under
-[filename]+hibernate-ogm-documentation/examples/gettingstarted+.
+A working example can be found in Hibernate OGM's source code under
+`documentation/examples/gettingstarted`.
+Check out the source code from https://github.com/hibernate/hibernate-ogm/[the GitHub repository]!
 
 What have we seen?
 
 * Hibernate OGM is a JPA implementation
   and is used as such both for mapping and in API usage
 * It is configured as a specific JPA provider:
-  [classname]+org.hibernate.ogm.jpa.HibernateOgmPersistence+
+  `org.hibernate.ogm.jpa.HibernateOgmPersistence`
 
-Let's explore more in the next chapters.
+To learn more about Hibernate OGM an all the NoSQL technologies it supports, see the http://hibernate.org/ogm/documentation/[Reference Documentation].
```

Comment:
This last line is the difference vs the example in the reference docs @gunnarmorling